### DISCUSSION TITLE
how can i tell chatgpt to not repeat himself? when i ask him something new

### DISCUSSION THREAD

**Discussion by JuliusCassy** [lencx/ChatGPT#1295] [Q&A] [answered]
pls help

**Comment by GylanSalih** [comment, marked as answer]
just open a new window oYou can try giving me more context or specifying what you're looking for in greater detail. If you've asked a similar question before, you can mention that and provide any additional information or nuances you'd like me to consider. This way, I can provide a more tailored response that hopefully avoids repetition.r tell ho
